.asf.yaml

```diff
@@ -50,7 +50,7 @@ github:
 
   enabled_merge_buttons:
     squash: true
-    merge: false
+    merge: true
```

Comment:
It probably needs to be changed and merged in a separate PR

Reply:
open a new [pr](https://github.com/apache/superset/pull/17560) to handle this.